__tests__/color-test.js

```diff
@@ -0,0 +1,72 @@
+const PPTX = require('../index.js');
+const fs = require('fs');
+const tmpDir = `${__dirname}/tmp`;
+
+describe('Presentation Module', () => {
+    describe('with an existing pptx file', () => {
+        test('should be able to add another slide, change the background color of each slide, and add a default text foreground color', async () => {
+            try {
+                expect.assertions(1);
+
+                if (fs.existsSync(`${tmpDir}/presentation-existing-non-default-colors.pptx`)) {
```

Comment:
should we clear all the files as part of the setup process like we do in the other tests?

Reply:
No, I hate it because I often unzip stuff to that directory, and then the test throws errors because it can't delete stuff that I have open. One of the reasons I made those smaller test cases to begin with.

Reply:
ok